src/main/java/seedu/address/model/tuitionclass/TuitionClass.java

```diff
@@ -113,6 +114,10 @@ public void addStudent(Name name) {
         studentNameList.add(name);
     }
 
+    public void addAllStudents(Name... names) {
+        studentNameList.addAll(Arrays.asList(names));
+    }
+
```

Comment:
you want to just overload addStudentList to also accept List<Name> so that don't have so many methods to keep track of?

Reply:
I think can arh but i think can merge this PR first